helm/cluster-vsphere/templates/helmreleases/cloud-provider-vsphere-helmrelease.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     giantswarm.io/service-priority: "{{ .Values.global.metadata.servicePriority }}"
     {{- include "labels.common" . | nindent 4 }}
 spec:
-  suspend: {{ include "isIpamSvcLoadBalancerEnabled" $ }} # It is unsuspended by the post-install/post-upgrade hook.
```

Comment:
this helmrelease was only suspended because it contained the kube-vip chart. with that being removed we don't need to suspend the CPI installation anymore